apiserver/facades/client/machinemanager/machinemanager.go

```diff
@@ -478,3 +479,38 @@ func (mm *MachineManagerAPI) removeUpgradeSeriesLock(arg params.UpdateSeriesArg)
 	}
 	return machine.RemoveUpgradeSeriesLock()
 }
+
+func (mm *MachineManagerAPI) validateSeries(argumentSeries, currentSeries string, machineTag names.MachineTag) error {
+	if argumentSeries == currentSeries {
+		return errors.Errorf("%s is already running series %s", machineTag, argumentSeries)
+	}
+	isOlderSeries, err := isSeriesLessThan(argumentSeries, currentSeries)
+	if err != nil {
+		return err
+	}
+	if isOlderSeries {
+		return errors.Errorf("machine %s is running %s which is a newer series than %s.", machineTag, currentSeries, argumentSeries)
+	}
+
+	return nil
+}
+
+// IsSeriesLessThan returns a bool indicating whether the first argument's
+// version is lexicographically less than the second argument's, thus indicating
+// that the series represents an older version of the operating system. The
+// output is only valid for Ubuntu series.
+func isSeriesLessThan(series1, series2 string) (bool, error) {
```

Comment:
Based on the capitalised name in the comment, I think you may have exported this in _export_test_ after-the-fact.
Don't think we need both `isSeriesLessThan` and `seriesLessThan`.